metrics/firebase/functions/test/main_test.dart

```diff
@@ -349,8 +295,6 @@ void main() {
     test(
       "does not create a task document if the build day data set successfully",
       () async {
-        final buildJson = testDataGenerator.generateBuildJson();
-
         whenDocumentSnapshotData().thenReturn(DocumentData.fromMap(buildJson));
```

Comment:
Maybe we can extract `DocumentData.fromMap(buildJson)` to a separate variable to avoid duplication?